shap/plots/_bar.py

```diff
@@ -315,6 +315,8 @@ def bar(shap_values, max_display=10, order=Explanation.abs, clustering=None, clu
     
     if show:
         pl.show()
+    else:
+        pl
```

Comment:
what the intention to return something here?

Reply:
yes, sorry, I had fixed and forgot to push. I did a force push after rebasing on top of the latest master

Reply:
Lao happy to add to other plots as well